lib/vsc/utils/generaloption.py

```diff
@@ -119,7 +119,11 @@ def get_empty_add_flex(allvalues):
 
     if empty is None:
         msg = "get_empty_add_flex cannot determine empty element for type %s (%s)"
-        self.log.raiseException(msg % (type(allvalues), allvalues))
+        msg = msg % (type(allvalues), allvalues)
+        if self is None:
+            raise Exception(msg)
+        else:
+            self.log.raiseException(msg)
```

Comment:
Make this exception a consistent type with whatever's chosen above (TypeError, I guess). So `except TypeError` works.